lom/src/lib/lomcommon/yang_parser/yang_prod_configs/device-health-actions-configs.yang

```diff
@@ -0,0 +1,130 @@
+module device-health-actions-configs {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/sonic-net/device-health-global-configs";
+
+    prefix dtm;
+
+    organization
+        "SONiC";
+
+    contact
+        "Device";
+
+    description
+        "Action specific configurations for Device health agent plugins";
+
+    revision 2023-06-28 {
+        description
+            "Initial revision.";
+    }
+	
+    grouping detection-action-config-common {
+        leaf Type {
+            type string;
+            description "Indicates Detection plugin";
```

Comment:
Not sure how to do... But here are the requirements
1. These are required in every action's config file.
2. You don't default here, but make it mandatory.
3. JSON is generated from every config file.

In the current model, if a safety check missed to update type, it will be a mis-config, because transparently it defaults to detection,